lib/src/main/java/com/auth0/android/lock/utils/CustomField.java

```diff
@@ -56,22 +58,66 @@
         int TYPE_EMAIL = 3;
     }
 
+    /**
+     * Location to store the field into.
+     */
+    @IntDef({PROFILE_ROOT, USER_METADATA})
+    @Retention(RetentionPolicy.SOURCE)
+    public @interface Storage {
```

Comment:
proposed "Storage" but can be named differently. Is like an interface users could use as "constant" when requiring this type of value. So users would pass `Storage.PROFILE_ROOT` instead of 0.

Should this be named something different?

Reply:
`Storage` makes sense to me in this context (and the accompanying docs change reads well with `Storage`)